__tests__/app.test.js

```diff
@@ -48,8 +48,10 @@ describe("GET /api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        expect(response.headers['content-type']).toEqual('application/json; charset=utf-8')
-        expect(response.body).toEqual(expect.any(Object))
+        expect(response.headers["content-type"]).toEqual(
+          "application/json; charset=utf-8"
+        );
+        expect(response.body).toEqual(expect.any(Object));
```

Comment:
You're not expecting the response to equal any object, you're expecting the response to equal the json file that you're updating.